src/components/drawer/drawer.scss

```diff
@@ -1,16 +1,12 @@
 .drawer {
   height: 100%;
   &__list-item {
+    overflow: visible;
```

Comment:
because of it we have ripple problem
![image](https://user-images.githubusercontent.com/14272396/37351546-01c7e386-26e4-11e8-8108-63daaf2ae11f.png)
without it popup doesn't work